banbot.js

```diff
@@ -292,19 +294,15 @@ bot.on('message', async ( message ) => {
             msg += '\n**RESULT:** 14 day suspension.'
         else if ( player.tier == 3 )
             msg += '\n**RESULT:** 30 day suspension.'
-        else if ( player.tier == 4 )
-            msg += '\n**RESULT:** 60 day suspension.'
-        else if ( player.tier == 5 )
-            msg += '\n**RESULT:** 90 day suspension.'
-        else if ( player.tier >= 6 )
+        else if ( player.tier >= 4 )
             msg += '\n**RESULT:** ' + target.displayName + ' banned from server. '
-        if ( !leftServer && target.roles.has( suspendedId ) && player.tier < 6 )
+        if ( !leftServer && target.roles.has( suspendedId ) && player.tier < 4 )
             msg += ' This is in addition to your current suspension.'
```

Comment:
NIT: you removed this type of context from other severity types. IMO, if you remove from one, remove from all, but I personally like how clear things are when these sorts of messages are left in.

Reply:
Had a quick look they are all still in place